### PR TITLE
implemented basic structs for VerifiableCredential

### LINKED ISSUE #8: Add basic verifiable credential model

### PR DESCRIPTION
**Pull request by woutslakhorst**
Closes #8

### CODE REVIEW

**Comment by woutslakhorst** [comment]
waiting for circleci config to be in master

File: vc.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	Proof              []interface{}		      `json:"proof"`
+}
+
+// CredentialStatus defines the method on how to determine a credential is revoked.
+type CredentialStatus struct {
+	ID                 url.URL                    `json:"id"`
+	Type			   string					  `json:"type"`
+}
+
+func (vc VerifiableCredential) Proofs() ([]Proof, error) {
```
Comment (reinkrul) [inline vc.go:39]: missing godoc

File: proof.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+// Proof represents a credential/presentation proof as defined by the Linked Data Proofs 1.0 specification (https://w3c-ccg.github.io/ld-proofs/).
+// The proof value must be implemented in a custom type since the specification doesn't define the json object for this.
+// For example: a jws for detached JSON Web Signatures uses the 'jws' json field
+type Proof struct {
+	// Type defines the specific proof type used.
+	// For example, an Ed25519Signature2018 type indicates that the proof includes a digital signature produced by an ed25519 cryptographic key.
+	Type			   string					  `json:"type"`
+	// ProofPurpose defines the intent for the proof, the reason why an entity created it.
```
Comment (reinkrul) [inline proof.go:14]: indenting is all screwed up?

File: vc.go [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+// VerifiableCredential represents a credential as defined by the Verifiable Credentials Data Model 1.0 specification (https://www.w3.org/TR/vc-data-model/).
+type VerifiableCredential struct {
+	// Context defines the json-ld context to dereference the URIs
+	Context            []URI                      `json:"context"`
+	// ID is an unique identifier for the credential. It is optional
+	ID                 *URI                        `json:"id,omitempty"`
+	// Type holds multiplte types for a credential. A credential must always have the 'VerifiableCredential' type.
+	Type			   []URI					  `json:"type"`
+	// Issuer refers to the party that issued the credential
+	Issuer			   URI 						  `json:"issuer"`
```
Comment (reinkrul) [inline vc.go:20]: indenting is all screwed up?

**Review by reinkrul — APPROVED** [review]
LGTM, except some weird indenting

File: vc.go (hunk 3 above)
Comment (woutslakhorst) [reply vc.go]: ran gofmt just now

**Pull request merged**